File: tests/07_phases_ompss_2.dg/c/success_lint_01.c
/*
<testinfo>
test_generator=config/mercurium-ompss-2
test_nolink=yes
</testinfo>
*/
int main() {
    int x, y, z = 0;
    #pragma oss lint in(x) out(y) inout(z)
    {
        x;
        y = 2;
        z++;
    }
}

File: tests/07_phases_ompss_2.dg/fortran/success_lint_01.f90
! <testinfo>
! test_generator=config/mercurium-ompss-2
! test_nolink=yes
! </testinfo>
PROGRAM P
    IMPLICIT NONE
    INTEGER :: X, Y, Z

    Z = 0

    !$OSS LINT IN(X) OUT(Y) INOUT(Z)
        PRINT *, X
        Y = 2
        Z = Z + 1
    !$OSS END LINT

END PROGRAM P
